Vagrantfile.erb

```diff
@@ -0,0 +1,133 @@
+<% config[:vagrantfiles].each do |vagrantfile| %>
```

Comment:
@therobot That isn't tracked in git (it's ignored local kitchen configs) so I'd just add Vagrantfile.erb to the gitignore in this commit to resolve that.